client/src/components/Lobby.jsx

```diff
@@ -19,26 +22,68 @@ export default class Lobby extends Component {
           const { username } = data;
           const socket = io();
 
+          // socket must be assigned before handleKeydown is useful
+          document.addEventListener('keydown', this.handleKeydown.bind(this));
+
           this.setState({
             name: username,
-            socket
+            socket,
           });
 
           socket.emit('join lobby', { name: username });
+          socket.on('lobby update', this.handleUpdateLobby.bind(this));
 
         } else {
-          this.props.history.replace("/login");
+          this.props.history.replace('/login');
         }
       })
       .catch((error) => {
-        console.log('Oh no! Are you sure you\'re logged in?');
+        console.log('Oh no! Are you logged in?');
       });
   }
 
+  handleKeydown(event) {
+    const { socket, lobby } = this.state;
+
+    switch (event.key.toLowerCase()) {
+    case 'w':
+      socket.emit('lobby move', { lobby: lobby, dir: 'up' });
+      break;
+    case 'a':
+      socket.emit('lobby move', { lobby: lobby, dir: 'left' });
+      break;
+    case 's':
+      socket.emit('lobby move', { lobby: lobby, dir: 'down' });
+      break;
+    case 'd':
+      socket.emit('lobby move', { lobby: lobby, dir: 'right' });
+      break;
+    }
```

Comment:
In your next PR, add support for the arrow keys also
(Ex. if (e.keyCode === 38) { // UP ARROW KEY)